atomic_reactor/plugins/pre_add_image_content_manifest.py

```diff
@@ -124,10 +123,12 @@ def layer_index(self):
         if self._layer_index is None:
             # Default layer index is 1, because base and 'FROM scratch' images
             #     *always* have 2 layers
+            # TODO: was this true for 'FROM scratch' images? Don't those have just 1 layer?
             self._layer_index = 1
```

Comment:
Does anyone know if this is true? I think scratch images get squashed to 1 layer, and since we will be converting custom base images to `FROM scratch` as well, those will get 1-squashed too.

Reply:
it is truth now, I am not sure what do we plan to do with podman and how that squashes

Reply:
Oh interesting, if you have a FROM scratch image then `podman build --squash` squashes it to 1 layer but imagebuilder squashes to 2 layers (the first one is an empty tarball). So we don't really know what the index will be, it depends on the build task implementation :thinking:

Reply:
yep

Reply:
Changed the index to 0 for now with a note about the potential issue with different build tasks